README.md

```diff
@@ -118,18 +118,7 @@ This test is still failing, even though the server is responding on localhost:80
 
 By default, web servers only serve content to web pages that were served by that web server. In order to allow other web pages, such as the ToDo Backend test page, to connect to the server, [Cross Origin Resource Sharing (CORS)](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) must be enabled.
 
-1. Open the Sources > Application > Application.swift file
-2. Add an import for the CORS library to the start of the file:
-   ```swift
-   import KituraCORS
-   ```
-3. Add the following into the start of the `postInit()` function:  
-   ```swift
-        let options = Options(allowedOrigin: .all)
-        let cors = CORS(options: options)
-        router.all("/*", middleware: cors)
-   ```
-4. Add the CORS library to the ToDoServer > Package.swift file
+1. Add the CORS library to the ToDoServer > Package.swift file
```

Comment:
Nit - this need to have a return at the end of the line (ie, two spaces in markdown). At the moment the next line is being appended to this one.

Reply:
Okay doing it.